pkg/operator/ceph/cluster/watcher.go

```diff
@@ -500,7 +505,11 @@ func concatenateWatcherIp(address string) string {
 	return watcherIP
 }
 
-func (c *clientCluster) createNetworkFence(ctx context.Context, pv corev1.PersistentVolume, node *corev1.Node, cluster *cephv1.CephCluster, cidr []string) error {
+func fenceResourceName(nodeName string, driver string) string {
+	return nodeName + "-" + driver
+}
+
+func (c *clientCluster) createNetworkFence(ctx context.Context, pv corev1.PersistentVolume, node *corev1.Node, cluster *cephv1.CephCluster, cidr []string, driver string) error {
```

Comment:
```suggestion
func (c *clientCluster) createNetworkFence(ctx context.Context, pv corev1.PersistentVolume, node *corev1.Node, cluster *cephv1.CephCluster, cidr []string) error {
```
we don't need `driver` as new parameter i ncreateNetworkFence, we can get via `pv.Spec.CSI.Driver` as we do at L535